src/components/nodes/hooks/useNodeStatus.ts

```diff
@@ -15,7 +15,7 @@ type NodeStatusDetail = {
 
 function getNodeStatusTitle(node: NodeStatusResult, t: VueI18nTranslation) {
   const i18n: Record<NodeStatus, string> = {
-    online: node.ping + ' ' + t('nodes.ms'),
+    online: node.ping + ' ',
```

Comment:
```suggestion
    online: node.ping,
```

Is the space still required?

Reply:
yes